src/python/grpcio/grpc/_channel.py

```diff
@@ -1270,11 +1402,14 @@ def channel_spin():
     channel_spin_thread.start()
 
 
-def _channel_managed_call_management(state):
+def _channel_managed_call_management(state: _ChannelCallState):
 
     # pylint: disable=too-many-arguments
-    def create(flags, method, host, deadline, metadata, credentials,
-               operationses, event_handler, context):
+    def create(flags: int, method: str, host: str, deadline: float,
+               metadata: Optional[MetadataType],
+               credentials: Optional[cygrpc.CallCredentials],
+               operationses: Iterable[Iterable[cygrpc.Operation]],
```

Comment:
Can this be `Sequence[Sequence[cygrpc.Operation]]`?

Reply:
`Iterable` might not be a `Sequence`, but looks like we're only use `for ... in ` for this Iterable so in this case should be fine to change it to Sequence of Sequence. Will also change the doc string too.

Reply:
This API is entirely internal, so we should be able to control what the call sites provide.